### DISCUSSION TITLE
Lab 1 Feedback

### DISCUSSION THREAD

**Discussion by jshaffer47** [nagrawal44/Interactive-Lab-Hub#1] [General]
Lab 1 Feedback

**Comment by jshaffer47** [comment]
The scene is a student studying (or someone working on something in general) and getting frustrated. The device is “mood activated.” Here is what I saw happening:

1. The device begins in a neutral state, indicated with blue light.
2. As the student gets more frustrated, the light gradually changes from blue to red.
3. The student sees the red light, allowing her to mindfully recognize that she needs to reduce her frustration in order to work more productively.
4. After meditating, the device reflects the fact that she is in a calmer state again by turning yellow and then green.
5. When the student returns to work, the device returns to a neutral state, indicated with blue light.

I thought this was pretty clear. My one suggestion is that maybe the device should flash to alert the user that they need to take a break (e.g. flash red).

**Comment by shivanidoshi26** [comment]
I think that if you're going to close your eyes and meditate, it'd be useful to have the device play a little sound to let you know that you're good to go back to work. That could be something to consider for the next version.

**Comment by HcwXd** [comment]
1. In the video I saw a device that uses different colors of light to indicate a user's emotions or feelings. When the light turns red, the user seems to be stressed and goes to do some meditation to relax.
2. I really like the idea of having a device that can show a user's stress levels or mood. It should be really helpful for people who really value their mental health or for people who want to understand themselves more in order to be more productive.
3. From the video, I'm curious about how many emotions or feelings can the device represent.
4. I'm also curious about how the device analyzes the users' feelings. Does it only use heart rate to tell whether a user is stressed?
